src/frontend/core/executor/impl/TriangleCountExecutor.cpp

```diff
@@ -548,7 +541,7 @@ long TriangleCountExecutor::getTriangleCount(int graphId, std::string host, int
             }
         }
 
-        triangleCount_logger.log("###COMPOSITE### Returning Total Triangles from executer ", "debug");
+        triangleCount_logger.info("###COMPOSITE### Returning Total Triangles from executer ");
```

Comment:
```suggestion
        triangleCount_logger.debug("###COMPOSITE### Returning Total Triangles from executor ");
```

Reply:
When it was `debug`, it did not appeared in the logs, which led to a confusion when reading the logs. That's why it was converted to `info`.

Reply:
I see, Ok, then lets keep it as info